index.js

```diff
@@ -339,3 +339,65 @@ module.exports.random = {
     return s * Math.pow(2, (-64 - e))
   }
 }
+
+/**
+ * Utils for simple signing and verification of HTTPS requests
+ **/
+module.exports.https = {
```

Comment:
please run the jsdoc npm script to update the docs https://github.com/brave/crypto/blob/master/package.json#L28